docs(clustered): warning for manual horizontal scaling (#5500)
influxdata/docs-v2: content/influxdb/clustered/reference/internals/storage-engine.md

### Horizontal scaling

{{% warn %}}

> You **must** only alter the number of replicas through the `AppInstance` to ensure
> that all of the appropriate variables for scaling IOx are handled.
> 
> Manually scaling resources may cause errors.

Reviewer: ```suggestion
#### Only use AppInstance to scale resources

Only alter the number of replicas through the `AppInstance` to ensure
that all of the appropriate variables for scaling resources are handled.
Manually scaling resources may cause errors.
```

{{% /warn %}}

Horizontal scaling (also known as "scaling out") involves increasing the number of
nodes or processes available to perform a given task.

> Horizontal scaling is typically used to increase the amount of workload or

Reviewer: Move the warning to after this paragraph.
